### PR TITLE
docs: Add `s3` config options to `serverless.yml.md`

### PR DESCRIPTION
**Pull request by nme077**
Closes: #9713

### CODE REVIEW

File: docs/providers/aws/guide/serverless.yml.md [hunk 1]
```diff
@@ -285,6 +285,11 @@ provider:
       format: '{ "requestId":"$context.requestId", "ip": "$context.identity.sourceIp", "requestTime":"$context.requestTime", "httpMethod":"$context.httpMethod","routeKey":"$context.routeKey", "status":"$context.status","protocol":"$context.protocol", "responseLength":"$context.responseLength" }'
 
     frameworkLambda: true # Optional, whether to write CloudWatch logs for custom resource lambdas as added by the framework
+  s3: # If you need to configure the bucket itself, you'll need to add s3 resources to the provider configuration: 
+    bucketOne: # Eventual additional properties in camel case
```
Comment (pgrzesik) [inline docs/providers/aws/guide/serverless.yml.md:289]: I believe it is worth mentioning that properties that are supported are the same ones as supported by CF resource for S3 bucket: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket.html

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thanks @nme077 :bow: Looks good, I only have one suggestion to include reference to CF documentation and we're good to go :100:

File: docs/providers/aws/guide/serverless.yml.md (hunk 1 above)
Comment (nme077) [reply docs/providers/aws/guide/serverless.yml.md:289]: Great suggestion. I have updated the docs.

**Comment by nme077** [comment]
@pgrzesik Thanks for the suggestion. I added that to the docs. Let me know if you need anything else.

**Comment by pgrzesik** [comment]
Thanks @nme077 - it looks great, but there's a failing formatting check - could you ensure to reformat the changes introduced with your commit? Thanks in advance :bow:

**Comment by nme077** [comment]
My apologies, I updated the formatting and ran the prettier check. Should be good to go!

**Comment by pgrzesik** [comment]
Hey @nme077, no worries :raised_hands: Unfortunatelly it looks like with fixing Prettier, your change that added documentation link was also reverted - is that the case or am I missing something?

**Comment by nme077** [comment]
@pgrzesik you are right! I added it back.

**Review by pgrzesik — APPROVED** [review]
thanks @nme077 :bow:

**Pull request merged**